PySDM/physics/adiabatic_exponent/moist_leading_terms.py

```diff
@@ -0,0 +1,18 @@
+"""
+adiabatic exponent, moist air, expanding to first order in qv, assuming qt=qv
+"""
+
+
+class MoistLeadingTerms:
+    def __init__(self, _):
+        pass
+
+    # pylint: disable=too-many-arguments
+    @staticmethod
+    def gamma(const, qv):
```

Comment:
@claresinger, let me suggest to rename `qv` to `vapour_mixing_ratio` (to disambiguate from specific_humidity)